### PR TITLE
Fixed breaking change with sending empty objects in a request

### PR DESCRIPTION
**Pull request by a-guiducci**
### What's Changed?

- list of changes
Updated request objects to not include empty objects when sending requests to DEID. This fixes a breaking change in client behavior that was introduced with the deid 3.6.0 release

Added a changelog to the repo to catalogue changes

Added integration tests to the test suite that covers a variety of calls and should help avoid future issues with releases

### PR Checklist

- [x] Updated unit tests
- [x] Ran unit tests

### CODE REVIEW

**Review by koryf — APPROVED** [review]
Can you remove the .gitignore newlines? 😄 
Other than that, thanks for adding the changelog, the integration tests and fixing the empty list handler!

File: src/privateai_client/tests/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+.env
```
Comment (a-guiducci) [inline src/privateai_client/tests/.gitignore:1]:
```suggestion
.env

```
